Automattic/woocommerce-payments
# Hide remember details checkbox on returning from platform checkout.

#### Changes proposed in this Pull Request
When a user returns to the merchant's checkout page by signout or guest checkout or back button, the `Remember your details` checkbox is visible there though this is a registered WooPay user. This PR fixes the issue by hiding the checkbox on returning from WooPay.

#### Testing instructions
- Add some products to the cart, and go to the checkout page.
- Use an email having a corresponding WooPay account.
- When OTP modal appears notice that the `Remember your details` checkbox is hidden.
- Complete OTP verification and go to platform checkout.
- From the `Contact` section click `Sign out` or `Checkout as guest` to return to the merchant checkout page.
- Ensure that the `Remember your details` checkbox is hidden.
